# remove /usr/local paths from all stuff thats used
# in Configure script, locincpth, loclibpth and glibpth are sedded
# afterwards not to conflict with the lib32 and lib64 patches

diff -rubB perl-5.8.8.orig/Configure perl-5.8.8/Configure
--- perl-5.8.8.orig/Configure	Tue Jan 15 10:41:29 2008
+++ perl-5.8.8/Configure	Tue Jan 15 10:44:59 2008
@@ -4424,7 +4422,7 @@
 : Set private lib path
 case "$plibpth" in
 '') if ./mips; then
-		plibpth="$incpath/usr/lib /usr/local/lib /usr/ccs/lib"
+		plibpth="$incpath/usr/lib /usr/ccs/lib"
 	fi;;
 esac
 case "$libpth" in
@@ -19144,7 +19142,7 @@
 	case "$ranlib" in
 	:) ranlib='';;
 	'')
-		ranlib=`./loc ranlib X /usr/bin /bin /usr/local/bin`
+		ranlib=`./loc ranlib X /usr/bin /bin`
 		$test -f $ranlib || ranlib=''
 		;;
 	esac
